conf/adzerk.py

```diff
@@ -30,6 +30,15 @@
                 "eventIds": [17, 20],
             }]
         }
+    },
+    # These are hardcoded and not expected to change.
```

Comment:
Where is this expectation coming from? Can you link to documentation stating this?